x-pack/plugins/ml/public/application/model_management/test_models/models/text_expansion/text_expansion_output.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type FC } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import {
+  EuiAccordion,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiSpacer,
+  EuiStat,
+  EuiTextColor,
+  EuiCallOut,
+} from '@elastic/eui';
+
+import { roundToDecimalPlace } from '@kbn/ml-number-utils';
+import { i18n } from '@kbn/i18n';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { TextExpansionInference, FormattedTextExpansionResponse } from '.';
+
+export const getTextExpansionOutputComponent = (inferrer: TextExpansionInference) => (
+  <TextExpansionOutput inferrer={inferrer} />
+);
+
+export const TextExpansionOutput: FC<{
+  inferrer: TextExpansionInference;
+}> = ({ inferrer }) => {
+  const result = useObservable(inferrer.getInferenceResult$(), inferrer.getInferenceResult());
+  if (!result) {
+    return null;
+  }
+
+  return (
+    <>
+      {result
+        .sort((a, b) => b.response.score - a.response.score)
+        .map(({ response, inputText }) => (
+          <>
+            <DocumentResult response={response} />
+            <EuiHorizontalRule />
+          </>
+        ))}
+    </>
+  );
+};
+
+export const DocumentResult: FC<{
+  response: FormattedTextExpansionResponse;
+}> = ({ response }) => {
+  const tokens = response.adjustedTokenWeights
+    .filter(({ value }) => value > 0)
+    .sort((a, b) => b.value - a.value)
+    .slice(0, 5)
```

Comment:
can we create a const for `5` to avoid magic numbers? I guess it could be `MAX_TOKENS`

Reply:
Updated in https://github.com/elastic/kibana/pull/159150/commits/faefba5fff6817c3b79e296a6749e1ed37bed2ef